runners/flink/pom.xml

```diff
@@ -26,8 +26,8 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
-  <artifactId>flink-parent</artifactId>
-  <version>0.2.0-incubating-SNAPSHOT</version>
```

Comment:
intended to remove version from parent and add it to children? Seems like this should be one of the inherited properties.